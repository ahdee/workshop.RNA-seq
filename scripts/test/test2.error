STAR version 2.5.3a loaded.
ngsutilsj version 0.4.7 loaded.
