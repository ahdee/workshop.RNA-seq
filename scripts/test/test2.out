Hello world
/asclab/users/leea3/workshop.RNA-seq/scripts/test
done!
